documentation/library_guide/dynamic_selection_api/fixed_resource_policy.rst

```diff
@@ -16,42 +16,31 @@ The fixed-resource policy always returns the same resource selection.
 .. code:: cpp
 
   namespace oneapi::dpl::experimental {
-  
-    template<typename Backend = sycl_backend>
-    class fixed_resource_policy {
-    public:
-      // useful types
-      using resource_type = typename Backend::resource_type;
-      using wait_type = typename Backend::wait_type;
-      
-      class selection_type {
+  template <typename ResourceType = sycl::queue, typename ResourceAdapter = oneapi::dpl::identity,
+          typename Backend = default_backend<ResourceType, ResourceAdapter>>
+    class fixed_resource_policy
+      : public policy_base<fixed_resource_policy<ResourceType, ResourceAdapter, Backend>, ResourceAdapter, Backend>

```

Comment:
```suggestion
  template <typename ResourceType = sycl::queue, typename ResourceAdapter = oneapi::dpl::identity,
            typename Backend = default_backend<ResourceType, ResourceAdapter>>
    class fixed_resource_policy
      : public policy_base<fixed_resource_policy<ResourceType, ResourceAdapter, Backend>,
                           ResourceAdapter, Backend>
```
To avoid horizontal scrolling in the code block.

Reply:
Addressed, and audited for all other instances of horizontal scrolling blocks